src/api/mongoClient/dbMigrate.ts

```diff
@@ -102,7 +102,7 @@ export async function migratePresets(db: Db) {
 
 export async function migrateMultiviewPresets(db: Db) {
   const multiviewPresets = await db
-    .collection<MultiviewPreset>('multiviews')
+    .collection<MultiviewPreset>('multiview-presets')
```

Comment:
Instead of doing this process again, is it possible to use one of the functions from the other files and use that here instead?